### PR TITLE
add grouped capistrano w/ docker

### PR DESCRIPTION
**Pull request by siepet**
Fixes a gemfile for dockerized dynamic staging with heaven.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -71,9 +68,15 @@ group :development do
   gem 'spring-commands-rspec'
 end
 
+group :development do
+  gem 'airbrussh', require: false
+  gem 'capistrano'
+  gem 'capistrano-rvm', require: false
```
Comment (rindek) [inline Gemfile]: This is not required for dockerized apps

File: Gemfile [hunk 2]
```diff
@@ -71,9 +68,15 @@ group :development do
...
+  gem 'capistrano-docker', github: 'netguru/capistrano-docker', tag: 'v0.2.5'
```
Comment (rindek) [inline Gemfile]: Remove tag & update gem :)

File: Gemfile (hunk 1 above)
Comment (siepet) [inline Gemfile]: It was already in file, i just changed the place. Should I remove it?

Comment (rindek) [inline Gemfile]: Yes please

Comment (siepet) [inline Gemfile]: Will do! :)

**Pull request closed without merging**